dashboard/app/models/plc/enrollment_unit_assignment.rb

```diff
@@ -68,6 +68,35 @@ def focus_area_positions
     plc_module_assignments.map{ |a| a.plc_learning_module.stage.position unless a.plc_learning_module.is_required? }.compact
   end
 
+  def summarize_progress
+    summary = []
+
+    categories_for_stage = plc_course_unit.script.stages.map(&:flex_category).uniq
+
+    if plc_course_unit.script.pd?
+      # Legacy PD courses don't have learning modules. So determine completeness by looking at stage completion
```

Comment:
At minimum, can we make this logic generic to all non-plc courses, rather than unique to pd courses?